frame/executive/src/lib.rs

```diff
@@ -332,6 +332,12 @@ where
 			);
 		}
 
+		frame_support::log::info!(
+			target: LOG_TARGET,
+			"try-runtime: Block #{:?} successfully executed",
```

Comment:
Just some ideas, we could do stuff here like:
- State trace (output of all modified storage keys)
- Print all executed calls + Events
- Consumed weight
- Block size etc
Probably most of it in JSON, so we can render it eventually (like chopsticks already does for state changes).

Reply:
Should this be done in this PR?

Reply:
No. Just some ideas for the future :smile: